src/api/java/com/minecolonies/api/entity/citizen/AbstractEntityCitizen.java

```diff
@@ -201,7 +201,7 @@ public void setTexture()
 
         final String textureBase = "textures/entity/" + model.getTextureBase() + (female ? "female" : "male");
         final int moddedTextureId = (textureId % model.getNumTextures()) + 1;
-        texture = new ResourceLocation(Constants.MOD_ID, textureBase + moddedTextureId + renderMetadata + ".png");
+        texture = new ResourceLocation(Constants.MOD_ID, textureBase + moddedTextureId + ".png");
     }
```

Comment:
We do need the renderMetadata for things like the logs in the lumberjack backpack, etc.

Reply:
Yeah, just saw that and was about to post here. It was auto-populating to 'null' for me for new citizens without any renderMetadata.  If you point me to where it is populated, I can try to make sure it gets defaulted to '' or something.

Reply:
setRenderMetadata is called by the AI

Reply:
turns out it doesn't actually break anything, reverted